### PR TITLE
rename Destination to Destinationpolicy, doc nits

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

File: proxy/v1/config/cfg.md [hunk 1]
```diff
@@ -210,15 +210,15 @@ Destination declares policies that determine how to handle traffic for a
...
 | ----- | ---- | ----- | ----------- |
 | destination | [string](#string) | optional | REQUIRED. Service name for which the service version is defined. The/ value MUST be a fully-qualified domain name,/ e.g. "my-service.default.svc.cluster.local". |
-| tags | [Destination.TagsEntry](#istio.proxy.v1alpha.config.Destination.TagsEntry) | repeated | Service version destination identifier for the destination service./ The identifier is qualified by the destination service name, e.g. version/ "env=prod" in "my-service.default.svc.cluster.local".N.B. The map is used instead of pstruct due to lack of serialization supportin golang protobuf library (see https://github.com/golang/protobuf/pull/208) |
+| tags | [DestinationPolicy.TagsEntry](#istio.proxy.v1alpha.config.DestinationPolicy.TagsEntry) | repeated | Service version destination identifier for the destination service./ The identifier is qualified by the destination service name, e.g. version/ "env=prod" in "my-service.default.svc.cluster.local".N.B. The map is used instead of pstruct due to lack of serialization supportin golang protobuf library (see https://github.com/golang/protobuf/pull/208) |
```
Comment (kyessenov) [inline proxy/v1/config/cfg.md]: Do you know why "/" show up in the text?

Comment (rshriram) [inline proxy/v1/config/cfg.md]: Frank asked the same.. Let me see..

**Comment by rshriram** [comment]
fixed.

**Comment by ZackButcher** [comment]
Github won't let me comment on chunks of the code that you didn't change in this PR, but can we delete the section at the bottom of the MD about scalar types? IDK why the docs generate it.

**Pull request merged**